docs/.gitignore

```diff
@@ -5,4 +5,6 @@ _site/
 generated/*.html
 
 # Ignore the sass cache
-.sass-cache
\ No newline at end of file
+.sass-cache
+
+_includes/auto_benchmarks.html
```

Comment:
Not required but would be good to add a comment why this file is ignored (even if it's rather obvious).